testapp/src/main/java/com/rakuten/tech/mobile/testapp/analytics/DemoAppAnalytics.kt

```diff
@@ -49,17 +49,14 @@ class DemoAppAnalytics private constructor(private val rasProjectId: String) {
         val cp = JSONObject()
             .put("mini_app_project_id", rasProjectId)
             .put("mini_app_sdk_version", BuildConfig.VERSION_NAME)
-        if (ratEvent.getMiniAppInfo() != null) {
-            cp.put("mini_app_id", ratEvent.getMiniAppInfo()?.id)
-                .put("mini_app_version_id", ratEvent.getMiniAppInfo()?.version?.versionId)
-        }
 
         return mapOf<String, Any>(
             "acc" to acc,
             "aid" to aid,
+            "etype" to ratEvent.getEvent().value,
             "actype" to ratEvent.getAction().value,
-            "pgn" to ratEvent.getLabel(),
-            "ssc" to "demoApp",
+            "target_ele" to ratEvent.getTargetElement(),
+            "pgn" to ratEvent.getPgn(),
```

Comment:
As mentioned in the other comment, you need to send `siteSection` as the separate field `ssc`.

Reply:
![Screen Shot 2021-08-30 at 12 30 36](https://user-images.githubusercontent.com/84305007/131281117-5cd90b0b-df9c-41cc-b14e-d9125aa3eb9b.png)
@corycaywood i think i misunderstood this doc, ok i will change it, thanks